subprojects/file-watching/src/main/java/org/gradle/internal/watch/registry/impl/NonHierarchicalFileWatcherUpdater.java

```diff
@@ -142,7 +142,7 @@ private void updateWatchedDirectories(Map<String, Integer> changedWatchDirectori
         if (watchedDirectories.isEmpty()) {
             LOGGER.info("Not watching anything anymore");
         }
-        LOGGER.info("Watching {} directories to track changes", watchedDirectories.entrySet().size());
+        LOGGER.debug("Watching {} directories to track changes", watchedDirectories.entrySet().size());
```

Comment:
Why make this into `DEBUG`?

Reply:
See the commit message:
> The watched directories change a lot on Linux (aka NonHierarchicalFileWatcherUpdater), so it doesn't makes sense to log something on info every time this happens.

We log the full number at the end of the build, that should be good enough.